http/src/main/resources/reference.conf

```diff
@@ -34,7 +34,7 @@ dataproc {
   }
   # This image is used for legacy jupyter image where hail is compatible with earlier version of spark
   legacyCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-dataproc-1-2-79-debian9-18fcafd"
```

Comment:
I'd love to remove support for Dataproc 1.2, and the legacy `leonardo-jupyter` image.. maybe will make a separate ticket for that as it seems some users are still using this image.